internal/installcfg/builder/builder_test.go

```diff
@@ -56,37 +56,37 @@ var _ = Describe("installcfg", func() {
 			ClusterNetworks:        []*models.ClusterNetwork{{Cidr: "1.1.1.0/24"}},
 			ServiceNetworks:        []*models.ServiceNetwork{{Cidr: "2.2.2.0/24"}},
 			MachineNetworks:        []*models.MachineNetwork{{Cidr: "1.2.3.0/24"}},
-			APIVip:                 "102.345.34.34",
-			IngressVip:             "376.5.56.6",
```

Comment:
Those are not even IP addresses, changed to a proper syntax matching the machine network